scripts/enable-datastore-viewer.sh

```diff
@@ -5,9 +5,10 @@
 # Enable the datastore viewer and reload nginx.
 
 ALLOW=""
-APPS_ID=""
+APP_ID=""
 IP="all"
 VIEWER_PORT="8099"
```

Comment:
AppScale opens up ports 8080-8099 for apps (https://github.com/AppScale/appscale/blob/master/firewall.conf#L25). 
That means that potentially all these ports can be occupied by running apps.

That block would have to be expanded for datastore viewer use.
Either with a static range 8100-8119 or dynamically after a datastore viewer is enabled.
In either case, the firewall.conf should be modified to allow this.